app/models/case/sar/offender.rb

```diff
@@ -108,9 +108,15 @@ def searchable_fields_and_ranks
   validate :validate_third_party_relationship
   validate :validate_third_party_address
 
+  validates :number_final_pages,
+            numericality: { only_integer: true, greater_than: 0,
+                            message: 'must be a positive whole number' }
+
+  validates :number_exempt_pages, 
+            numericality: { only_integer: true, greater_than: 0,
+                            message: 'must be a positive whole number' }
+
```

Comment:
hmm very interesting, never know rails can have such feature :),  only thing I am thinking  "only_integer" is necessary here as we have defined the type is integer,  seems to me it is redundant one.

Reply:
I think this will throw an error from to the form if numbers with decimals are submitted 👍